Gemfile.lock

```diff
@@ -809,7 +809,7 @@ DEPENDENCIES
   web-console (~> 3.5)
 
 RUBY VERSION
-   ruby 2.6.6p146
+   ruby 2.6.5p114
```

Comment:
```suggestion
   ruby 2.6.6p146
```

Reply:
@mrcasals @oriolgual this is still pending, I'm applying the suggestion